src/sentry/static/sentry/app/components/radio.tsx

```diff
@@ -1,27 +1,33 @@
 import React from 'react';
 import {css} from '@emotion/core';
+import isPropValid from '@emotion/is-prop-valid';
 import styled from '@emotion/styled';
 
 import {growIn} from 'app/styles/animations';
 import {Theme} from 'app/utils/theme';
 
-type CheckedProps = React.HTMLProps<HTMLInputElement> & {theme: Theme};
+type CheckedProps = React.HTMLProps<HTMLInputElement> & {
+  theme: Theme;
+  radioSize?: 'small';
+};
 
 const checkedCss = (p: CheckedProps) => css`
   display: block;
-  width: 1rem;
-  height: 1rem;
+  width: ${p.radioSize === 'small' ? '8px' : '1rem'};
+  height: ${p.radioSize === 'small' ? '8px' : '1rem'};
   border-radius: 50%;
   background-color: ${p.theme.active};
   animation: 0.2s ${growIn} ease;
   opacity: ${p.disabled ? 0.4 : null};
 `;
 
-const Radio = styled('input')`
+const shouldForwardProp = p => p !== 'radioSize' && isPropValid(p);
```

Comment:
should we move it to the Radio or rename it to something like  `shouldRadioForwardProps`?

Reply:
Going to remove `shouldForwardProp` as it should work without it.